test/unit/data/json/undefined_succeeded_pending_scenarios_specified_v3.json

```diff
@@ -0,0 +1,229 @@
+[
```

Comment:
Added a new testcase to check the feature status correctly for statuses `undefined`, `pending`, `passed`